SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/users.md

To retrieve the users in an enterprise, call
[`client.users.listForEnterprise(filterTerm:fields:offset:limit:)`][get-enterprise-users].

> The method returns an iterator object in the completion that you can use to get the users.

Reviewer: ```suggestion
The method returns an iterator object in the completion, which is used to get the users.
```
